classes/class06/sessions/app.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var session = require('express-session');
+var path = require('path');
+
+var logger = require('morgan');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
+
+var app = express();
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(session({
+	secret: 'secret',
+	resave: false,
+	saveUninitialized: true
+}));
+
+app.get('/', function(req, res) {
+	console.dir(req.cookies);
+	console.dir(req.session);
+	
+	if (req.session.counter)
+		req.session.counter++;
+	else
+		req.session.counter = 1;
+
+	res.send('hello');
```

Comment:
Could you send the counter itself or "Hello, were glad to see you've returned "+counter+" times!" to make it a bit easier to see without requiring the cookie viewing in the browser?